### ISSUE TITLE
Queries on applying ONCOCNV on Exome Seq Data

### ISSUE THREAD

**Issue by vyomeshj209** [BoevaLab/ONCOCNV#14]
Hello Sir/Ma'am,

1] Trying to apply oncocnv for exome data i.e. without reference to amplicon id.  What should be modification required in bed file for applying it on Exome data.

Standard bed file:
track name="4477685_CCP_Designed" description="Amplicon_Insert_4477685_CCP" type=bedDetail					
chr1	2488068	2488201	AMPL242431688	0	TNFRSF14

Bed file used for Exome data:
track name="Covered" description="Illumina Exon - Genomic regions covered by probes" db=hg19			
chr1	14694	14814  AMPL1354  0   WASH7P

Above bed file used, contains random amplicon id in 3rd column and 0 in 5th column, is it right strategy to use for Exon seq data?

2] While applying ONCOCNV for EXON data what precautions or prerequisite should be taken care of?

Kindly Guide.

Regards,
Vyomesh

**Comment by valeu** [comment]
Dear Vyomesh,
yes, your bed file should work. Two important considerations: Amplicon IDs should be unique, and Gene names should be provided to allow prioritizing breakpoints in intergenic regions.

Otherwise, it looks good to me. 
Of note, OncoCNV can take some time to run on exome-seq as it was designed for amplicon-seq with fewer regions.
Also, OncoCNV will not take into account BAF info from SNPs. If you wish to do so, you can try the ControlFREEC method (C++) also developed by my team for whole genome and whole exome sequencing data.

**Comment by vyomeshj209** [comment]
Hi Valentina Boeva,

We are currently using ControlFREEC method  & reproduce the same results in ONCOCNV. But we are finding difference in outcome.

Test Sample (Output from ControlFEEEC):
19	66376	5610130	8	gain	STK11

Test Sample (Output from ONCOCNV):
chr19	1206907	1207207	STK11	AMPL224571	0.669761037
chr19	1218389	1218509	STK11	AMPL224572	-0.475328357
chr19	1219292	1219412	STK11	AMPL224573	0.052617935
chr19	1220372	1220522	STK11	AMPL224574	-0.253691951
chr19	1220573	1220723	STK11	AMPL224575	-0.179748549
chr19	1221199	1221332	STK11	AMPL224576	0.493683735
chr19	1221901	1222021	STK11	AMPL224577	-0.111910189
chr19	1222975	1223181	STK11	AMPL224578	-0.059582191
chr19	1223609	1223760	STK11	AMPL224579	NA
chr19	1226445	1226656	STK11	AMPL224580	-0.255746161

Can you help in understanding why there is difference? 

Thanks,
Vyomesh

**Comment by valeu** [comment]
Dear Vyomesh, do you have .png images with the output of FREEC and ONCOCNV for this gene?
When you run FREEC, do you provide the .bed file with exon coordinates?

**Comment by vyomeshj209** [comment]
Hi Valentina Boeva,

Apologies for late response.
Yes, using bed file with exon coordinates for running  FREEC tool.

For FREEC, I do not have image but for ONCOCNV the image attached below:
![image](https://user-images.githubusercontent.com/77434374/152986950-52e90c2c-1f08-4515-96bc-ef31cfcd864f.png)

**Comment by valeu** [comment]
I am lost.. I thought you had WES data.. But then why do you have so few regions in the OncoCNV output figure?

**Comment by valeu** [comment]
For FREEC, you can generate the .png using an R script in the scripts folder

**Comment by vyomeshj209** [comment]
Yes working on WES but have restricted gene list to approx. 1200 gene.  Will try the R script for FREEC tool and share the details
